compiler/rustc_errors/src/lib.rs

```diff
@@ -1004,7 +1001,6 @@ impl DiagCtxt {
     ) -> ErrorGuaranteed {
         let treat_next_err_as_bug = self.inner.borrow().treat_next_err_as_bug();
         if treat_next_err_as_bug {
-            // FIXME: don't abort here if report_delayed_bugs is off
```

Comment:
BTW, I didn't understand this FIXME comment. "report_delayed_bugs is off" is now always true, but I think we still need to abort here. Was the comment wrong?